tools/test-proxy/Azure.Sdk.Tools.TestProxy/Startup.cs

```diff
@@ -67,6 +67,10 @@ public static void Main(bool insecure = false, string storageLocation = null, st
                 Environment.Exit(0);
             }
 
+            GitProcessHandler gph = new GitProcessHandler();
+            // This throws and will exit
+            gph.VerifyGitMinVersion();
```

Comment:
```suggestion
            new GitProcessHandler().VerifyGitMinVersion();
```

Reply:
Will be updated to the suggested change with the next commit.